CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
-## 2.8.0 (Unreleased)
+## 2.9.0 (unreleased)
+
+* Improved tags isolation [#320](https://github.com/vmware/go-vcloud-director/pull/320)
+* Added command `make tag` to check tags isolation tests [#320](https://github.com/vmware/go-vcloud-director/pull/320)
```

Comment:
Should be `tagverify` and also need a word "**testing** tags isolation" in the first changelog.

Reply:
fixed